src/service/build-content-items.ts

```diff
@@ -110,3 +111,30 @@ export const mapArticlesToContentItems = (
 
     return items.filter(item => PillCategoryValues.includes(item.category));
 };
+
+export const mapCommunityEventsToContentItems = (
+    allCommunityEvents: CommunityEvent[]
+) => {
+    const items: ContentItem[] = [];
+    allCommunityEvents.forEach((event: CommunityEvent) => {
```

Comment:
nitpick: i think you can just use `map` here instead of initializing an `items` array. 
```
return allCommunityEvents.map((event: CommunityEvent) => ({ ... }))
```
would make it slightly more efficient + matches the function name better